src/libteam/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+## This script is to build libteam
+##
+## USAGE:
+##   ./build.sh
+
+# Obtain libteam
+rm -rf ./libteam
+git clone https://github.com/jpirko/libteam.git
+pushd ./libteam
+git checkout -f 8c1cea0
+popd
+
+git clone https://anonscm.debian.org/git/collab-maint/libteam.git tmp
+pushd ./tmp
+git checkout -f da006f2
```

Comment:
there is no need to clone github libteam. you can use uscan to download the source code.
see here. https://wiki.debian.org/debian/watch